canosp2020/preprocessing.py

```diff
@@ -54,10 +61,15 @@ def _custom_op(doc: doc.Doc, context=None, settings=None, **kwargs):
 
         # default clean_text method, strip HTML tags and lower case
         raw_text = doc.clean.lower()
-        spacy_doc = spacy_nlp(raw_text)
+
+        # remove stopwords
+        stop_words = stopwords.words("english")
```

Comment:
We are already removing stop words at https://github.com/mlopatka/CANOSP2020/pull/81/files#diff-c56f45d7c60bf7afbaf1092a1edcc0e1R63